### PR TITLE
feature/#CU-h4vbmt-Additional-Baggage---MCFS-doc

### PR DESCRIPTION
**Pull request by niloufarmp**
**Added additional bags price to the price confirm response:**
image
image

**Added additional bags to the book request:**
image
image

**Added additional bags to the booking details response(for each passenger):** (Same structure as in book request)
image

**Added additional bags to the booking details response(pricing):** (Same structure as in book request)
image

### CODE REVIEW

File: tn_api_pricing_booking_spec.yml [hunk 1]
```diff
@@ -402,10 +402,50 @@ components:
                 type: string
                 description:
                 example: "1"
-            baggage_cost:
```
Comment (mcclymont-k) [inline tn_api_pricing_booking_spec.yml:409]: Why did we remove baggage cost? Was this always empty?

**Review by mcclymont-k — APPROVED** [review]

**Comment by mcclymont-k** [comment]
Its also in booking detail response itinerary.segments.baggage - I'm not sure it should be there aye?

File: tn_api_pricing_booking_spec.yml [hunk 2]
```diff
@@ -406,18 +406,54 @@ components:
...
-                type: float
-                description:
-                example: 30.00
+            additional_checked_bags:
+                type: array
+                items:
+                  type: object
+                  description: Excess Baggage
```
Comment (Iceman997) [inline tn_api_pricing_booking_spec.yml]:
```suggestion
                  description: Additional Checked Baggage
```

File: tn_api_pricing_booking_spec.yml [hunk 3]
```diff
@@ -406,18 +406,54 @@ components:
...
+                      example: 40.00
+                    restriction:
+                      type: string
+                      description: Information about baggage restrictions
+                      example: UPTO50LB/23KG AND UPTO62LI/158LCM
+            additional_carry_on_bags:
+                type: array
+                items:
+                  type: object
+                  description: Excess Baggage
```
Comment (Iceman997) [inline tn_api_pricing_booking_spec.yml]:
```suggestion
                  description: Additional Carry On Baggage
```

**Comment by niloufarmp** [comment]
> baggage

oh no it shouldn't. I'm gonna separate the schemas

**Comment by niloufarmp** [comment]
fixed & ready for review again

**Review by mcclymont-k — APPROVED** [review]

**Review by Iceman997 — APPROVED** [review]

**Comment by trip-ninja-inc** [comment]
Task linked: [CU-h4vbmt Additional Baggage - MCFS doc](https://app.clickup.com/t/h4vbmt)

**Pull request merged**